generated/installer/ibm-spectrum-scale-csi-operator.yaml

```diff
@@ -65,7 +65,11 @@ spec:
         - mountPath: /tmp/ansible-operator/runner
           name: runner
           readOnly: true
-      - env:
+        env:
+        - name: CSI_DRIVER_IMAGE
+          value: quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-driver:v1.1.0 
```

Comment:
Does this become another place where the driver image is baked into operator in addition to "roles/csi-scale/defaults/main.yml". How does the end user set  CSI_DRIVER_IMAGE?

Reply:
No, this  is the solution to the bake in, now it's in a non compiled file  that we can  generate without an image redeploy.